modules/apps/transfer/keeper/msg_server.go

```diff
@@ -39,5 +40,5 @@ func (k Keeper) Transfer(goCtx context.Context, msg *types.MsgTransfer) (*types.
 		),
 	})
 
-	return &types.MsgTransferResponse{}, nil
+	return &types.MsgTransferResponse{Sequence: sequence}, nil
```

Comment:
Could you assert the res.Sequence is non 0 on passing and 0 on failing tests (for msg_server_test.go)?

Reply:
sure! updated.